src/styles/icons/arrow left.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="15" height="24" viewBox="0 0 15 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
There's a space in the name of the files for the arrows that _might_ cause issues when importing it into different files. I'd suggest adding a dash to replace the space in case!

Reply:
Nice catch! Updated all oof the icons to have dashes instead of spaces!